### DISCUSSION TITLE
UNREG and MainChat

### DISCUSSION THREAD

**Discussion by Boca-NeedM** [luadch/luadch#137] [General]
etc_msgmanager_permission_main = {  -- who is allowed to write main messages? (array of boolean)

        [ 0 ] = false,  -- unreg
        [ 10 ] = false,  -- guest
        [ 20 ] = true,  -- reg
        [ 30 ] = true,  -- vip
        [ 40 ] = true,  -- svip
        [ 50 ] = true,  -- server
        [ 55 ] = true,  -- sbot
        [ 60 ] = true,  -- operator
        [ 70 ] = true,  -- supervisor
        [ 80 ] = true,  -- admin
        [ 100 ] = true,  -- hubowner
    },

Above Does not work..  
PM however does work, UNREG User cannot PM.

**Comment by pulsar-de** [comment]
Sry but i can't reproduce this problem. I tried to login as UNREG (with your config above), and when i try to write in main i got:

`<[BOT]HubSecurity> You are not allowed to write messages in main chat.`
